### PR TITLE
feat: update workspace with more information on team plan

### LINKED ISSUE #264: Feature: Add explanation about changing workspaces
### Type of feature

🍕 Feature

### Current behavior

When you change workspaces, you'll only see the insights of that specific workspace.

### Suggested solution

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by BekahHW**
## Description

Add feature to share workspaces and team workspaces in OpenSauced app:

Users can now easily share their workspace by copying the URL. Team workspaces, available in the PRO plan, allow teams to collaborate on projects privately. Upgrade to a team workspace for unlimited insights and control over visibility. Additionally, users can add members to their workspace and manage their permissions.

_Generated using [OpenSauced](https://opensauced.ai/)_
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

closes #264 #256 
## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: docs/features/workspaces.md [hunk 1]
```diff
@@ -76,3 +77,24 @@ Start by creating a workspace name, then click "Add repositories." You'll have t
...
+Anything you create in a workspace is specific to that workspace. When you switch workspaces, you'll see the repositories and contributors specific to that workspace.
+
+## How to Share a Workspace
+
+You can share your workspace with others by copying the URL and sending it to them. If your workspace is private (a PRO plan feature), only those with access to the workspace will be able to view it.
+
+
+## Team Workspaces
+
+Team workspaces are part of the PRO plan and are designed to help teams collaborate on and understand open source projects. Team workspaces are public by default. You can switch it to private in your settings so only members of the team can view the workspace.
```
Comment (adiati98) [inline docs/features/workspaces.md:92]: I think a slight of wording format change here would be better for clarity.

```suggestion
Team workspaces are part of the PRO plan and are designed to help teams collaborate on and understand open source projects. By default, team workspaces are public. However, you can switch it to private in your settings so only the team members can view it.
```

File: docs/features/workspaces.md [hunk 2]
```diff
@@ -76,3 +77,24 @@ Start by creating a workspace name, then click "Add repositories." You'll have t
...
+
+By upgrading to a team workspace, you'll have unlimited contributor and repository insights, as well as the ability to toggle between private and public visibility. 
+
+You can add someone to your workspace by clicking on the "Edit" button on the top right of the workspace dashboard and then entering the username of the workspace member you would like to add.
```
Comment (adiati98) [inline docs/features/workspaces.md:96]:
```suggestion
You can add someone to your workspace by clicking on the "Edit" button at the top right of the workspace dashboard, then enter the username of the member you would like to add.
```

**Review by adiati98 — APPROVED** [review]
@BekahHW just a couple of nitpicks here.
But they don't block this PR. 😊

File: docs/features/workspaces.md (hunk 1 above)
Comment (BekahHW) [reply docs/features/workspaces.md:92]: I think it's better to start with the subject of the sentence. So Workspaces are private by default is preferred in this instance. But I did adjust structure based on your suggestions.

**Pull request merged**